docs/src/frontend/index.md

```diff
@@ -444,6 +444,18 @@ acc.getAddress() => string
 
 Returns the account's address as a string. The format of this string varies across connectors.
 
+Example:
+
+```js
+load: /examples/interact-this/index.mjs
+md5: 34eaa215660697ce8c5ba87387c8f0f1
+range: 14-19
+```
+
+In this code snippet, `{!js} getAddress` is being called in order to obtain the individual addresses of `Alice` and `Bob` that were created with the `{!js} newTestAccounts` code.
```

Comment:
We can tighten up the language:
In this snippet, `{!js} getAddress` obtains the addresses of `Alice` and `Bob`.
Next, each participant's address is logged.

Reply:
I think simplifying it that much makes it a little more difficult for newer programmers to understand, but I agree it is a bit wordy. How about this:

In this snippet, `{!js} getAddress` obtains the individual addresses of `Alice` and `Bob` created by `{!js} newTestAccounts`.
Next, each participant's address is logged.

Reply:
Sounds good. 
You'll need to rebase, also.